registers/TUTORIAL.md

```diff
@@ -14,3 +14,45 @@ You can run it with
 
 By default it uses the "register" register (a list of all registers)
 but you can point it at others with the ```--register``` flag.
+
+Step 2
+------
+
+Next, we are going to put the data into a Trillian log. The first step
+is run the log server and create a log.
+
+Build Trillian:
+
+```make trillian```
+
+Run the log server (this will tie up a terminal window):
+
+```make tlserver```
+
+And create a log (if you need to redo this, you'll need to delete
+logid):
+
+```make createlog```
+
+So now we have a log ready for entries from the register. To fill that
+log, we connect to the log using ```grpc.Dial()```, then create a new
```

Comment:
This is a little jarring - up until here we've been doing stuff in the terminal, but we switched without warning to golang code - maybe just set the stage a little?

"To fill that log, our application will connect to the log using ..."?

Also, single backticks :)